index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <header class="header-logo">
+        <h2 class="logo">My Logo</h2>
+        <img src="images/menu-rounded.png" alt="" class="image-icon"> 
+         
+    </header>
```

Comment:
- [ ] Good job already @MarkYegon7, Here please make your `logo` a link item with an anchor `a` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- [ ] Also, The hamburger button should be wrapped inside a `nav` tag. The `nav` tag is for semantics reasons. And remember in the future your navigation link are to go here.
- [ ] Besides that, Please make sure to use the right font for your logo. in this case `font-family: 'Poppins'`. and the appropriate color and size for the hamburger button.

### Figma design
![Screenshot 2023-02-17 224327](https://user-images.githubusercontent.com/69077061/219774114-c9d4e34f-e42c-43a6-86cf-4d8ea0d8a245.png)

### Your design
![image](https://user-images.githubusercontent.com/69077061/219772953-b036e176-fd81-4192-9f92-5fc3d52e4c60.png)